extension/popup/popupView.jsx

```diff
@@ -684,7 +684,6 @@ class TypingInput extends PureComponent {
           autofocus="1"
           onKeyPress={this.onInputKeyPress}
           onChange={this.onInputTextChange}
-          defaultValue={this.value}
```

Comment:
This is here so that the first keypress gets entered into the input field, so removing it will cause problems with typed input.

The problem is that this is how onInputStarted is intended to work:

```js
  onInputTextChange = event => {
    this.value = event.target.value;
    if (this.value && this.props.onInputStarted) {
      this.props.onInputStarted(this.value);
    }
  };
```

But when clicked it's used like this:

```js
<button className="type-input-button" onClick={onInputStarted}>
```

In that case `onInputStarted` is called with an event, which is an object. Instead it needs to be something like `onClick={() => onInputStarted("")}`

Reply:
Alright, it makes sense. Fixing that now.